cpp/ps0000/main.cpp

```diff
@@ -1,16 +1,9 @@
-/******************************************************************************
+#include <iostream> // 2/2
```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`